tests/dicts/keypath/test_keypath_dict_list_wildcard.py

```diff
@@ -593,6 +612,7 @@ def test_complex_wildcard_non_consecutive(self):
                 }
             ],
         }
```

Comment:
Could you add more than one item to each list accessed using the `*`?

Reply:
working on that. While I was adding more items I added tests for ```pop()``` as well. There is a bug. Trying to solve it. This rabbit hole never ends.

Reply:
This feature is very very cool, but it can have many drawbacks on other methods, for this reason it must be tested very well in more context/scenarios as possible. The rain of issues is always ready.